build.zig

```diff
@@ -17,10 +17,54 @@ pub fn build(b: *std.Build) !void {
         },
     };
 
+    const c_sdl_preferred_linkage = b.option(
+        std.builtin.LinkMode,
+        "c_sdl_preferred_linkage",
+        "Prefer building statically or dynamically linked libraries (default: static)",
+    ) orelse .static;
+    const c_sdl_strip = b.option(
+        bool,
+        "c_sdl_strip",
+        "Strip debug symbols (default: varies)",
+    ) orelse (optimize != .Debug);
+    const c_sdl_sanitize_c = b.option(
+        enum { off, trap, full }, // TODO: Change to std.zig.SanitizeC after 0.15
+        "c_sdl_sanitize_c",
+        "Detect C undefined behavior (default: varies)",
+    ) orelse .off;
+    const c_sdl_pic = b.option(
+        bool,
+        "c_sdl_pic",
+        "Produce position-independent code (default: varies)",
+    ) orelse true;
+    const c_sdl_lto = b.option(
+        enum { true, false, none, full, thin }, // TODO: Change to std.zig.LtoMode after 0.15
```

Comment:
```suggestion
        enum { none, full, thin }, // TODO: Change to std.zig.LtoMode after 0.15
```

You can remove true/false, I only kept those for backward compatibility.